liberty-maven-plugin/src/it/verify-user-feature-it/postbuild.groovy

```diff
@@ -0,0 +1,10 @@
+import java.io.File;
+
+File buildLog = new File( basedir, 'build.log' )
```

Comment:
What is this groovy file doing in LMP?

Reply:
This is a post build script that runs as part of Maven invoker plugin. I check expected warning messages here. More info about post build script can be found [here](https://maven.apache.org/plugins/maven-invoker-plugin/examples/post-build-script.html)